apps/condo/domains/organization/access/OrganizationLinkEmployeeAccess.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Generated by `createschema organization.OrganizationLinkEmployeeAccess 'link:Relationship:OrganizationLink:CASCADE; employee:Relationship:OrganizationEmployee:CASCADE; canManageOrganization:Checkbox; canManageEmployees:Checkbox; canManageRoles:Checkbox; canManageIntegrations:Checkbox; canManageProperties:Checkbox; canManageTickets:Checkbox;'`
+ */
+
+const { OrganizationEmployee } = require('../utils/serverSchema')
+const { OrganizationLink } = require('../utils/serverSchema')
+
+async function canReadOrganizationLinkEmployeeAccesses ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin || user.isSupport) return true
+}
+
+async function canManageOrganizationLinkEmployeeAccesses ({ authentication: { item: user }, originalInput, operation, itemId, context }) {
+    if (!user) return false
+    if (user.isAdmin || user.isSupport) {
+        // checking that the employee is in the organization "from"
+        const { employee : { connect: { id: employeeFromId } }, link: { connect: { id: linkId } } } = originalInput
```

Comment:
What about `disconnect`, that will be presented, if by updating we would like to revoke employee access?

Reply:
There won't be disconnect, but yes, this code didn't cover update. Fixed in ce3c5cb